src/scancode/cli.py

```diff
@@ -216,6 +239,8 @@ def validate_formats(ctx, param, value):
               help=('Set <output_file> format <style> to one of the standard formats: %s '
                     'or the path to a custom template' % ' or '.join(formats)),
               callback=validate_formats)
+@click.option('--ignore', default=None, multiple=True, metavar='<pattern>',
+              help=('Ignore files mentioned in the GLOB <pattern> while scanning. Check --examples for more information'))
```

Comment:
What about this:
`Ignore files matching <pattern>.`
The check example part is not needed.